src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -94,7 +94,7 @@ class SignInModal extends React.Component {
           [EXTERNAL_SERVICES.dslogon],
           'You may have trouble signing in with DS Logon',
           'warning',
-          'We’re sorry. We’re working to fix some problems with our DS Logon sign in process. If you’d like to sign in to VA.gov with your DS Logon account, please check back later.',
+          "We’re sorry. We’re working to fix some problems with our DS Logon sign in process. You can sign in to VA.gov with an existing ID.me account or you can create an account and verify your identity through ID.me. Learn how to create an account through ID.me. If you continue to have trouble, please call the DS Logon help desk at (800) 538-9552.",
```

Comment:
```suggestion
          <>
            <p>We’re sorry. We’re working to fix some problems with our DS Logon sign in process. You can sign in to VA.gov with an existing ID.me account or you can create an account and verify your identity through ID.me.</p>
            <p><a href="...">Learn how to create an account through ID.me.</a></p>
            <p>If you continue to have trouble, please call the DS Logon help desk at (800) 538-9552.</p>
          </>,
```

Not sure where the line break is or what URL you want to link to, but hopefully you can follow this as a guideline and get the rendered output you wanted.